package.json

```diff
@@ -785,6 +785,28 @@
           "command": "confluent.topic.consume.duplicate",
           "group": "navigation",
           "when": "activeWebviewPanelId == message-viewer"
+        },
+        {
+          "command": "confluent.diff.selectForCompare",
+          "when": "resourceScheme in confluent.diffableResources",
+          "group": "3_compare"
```

Comment:
No idea why one section uses `3_compare` and another uses `1_diff`, but they're based on info found in https://code.visualstudio.com/api/references/contribution-points#Sorting-of-groups